students/jakielczyk_miroslaw/lesson_05_lists/greater_than_neighbours.py

```diff
@@ -0,0 +1,7 @@
+spam_list = input().split(' ')
+count = 0
+for x in range(1, len(spam_list) - 1):
+    if (int(spam_list[x]) - int(spam_list[x - 1])) > 0 \
```

Comment:
good multiline if.
my personal preference would be to put `and` at the end of line 4 instead of begin of line 5.